File: Exemplos/App_IntentDigitalHUB_Kotlin/Elgin_IntentDigitalHUB_Java-font/app/src/main/java/com/elgin/intent_digital_hub/IntentDigitalHubService/TERMICA/ImprimeXMLSAT.kt
package com.elgin.intent_digital_hub.IntentDigitalHubService.TERMICA

import com.elgin.intent_digital_hub.IntentDigitalHubService.TERMICA.TermicaCommand

class ImprimeXMLSAT(private val dados: String, private val param: Int) :
    TermicaCommand("ImprimeXMLSAT") {
    override fun functionParameters(): String {
        return "\"dados\"" + ":" + "\"" + dados + "\"" + "," +
                "\"param\"" + ":" + param
    }
}

File: Exemplos M10/App_IntentDigitalHUB_Flutter/Elgin_IntentDigitalHUB_Flutter-font/lib/IntentDigitalHubService/TERMICA/imprime_xml_sat.dart
import 'package:intent_digital_hub/IntentDigitalHubService/TERMICA/termica_commad.dart';

class ImprimeXMLSAT extends TermicaCommand {
  final String _dados;
  final int _param;

  ImprimeXMLSAT(this._dados, this._param) : super('ImprimeXMLSAT');

  @override
  functionParameters() {
    return '"dados"'
        ':'
        '"'
        '$_dados'
        '"'
        ','
        '"param"'
        ':'
        '$_param';
  }
}
